cdap-master/pom.xml

```diff
@@ -964,6 +965,11 @@
           <groupId>org.apache.logging.log4j</groupId>
           <artifactId>log4j-to-slf4j</artifactId>
         </dependency>
+        <dependency>
+          <groupId>commons-io</groupId>
+          <artifactId>commons-io</artifactId>
+          <version>${commons-io.version}</version>
+        </dependency>
```

Comment:
Can we just do this in `cdap-common` directly where we have `hadoop-common` as dependency instead of overriding here?

Reply:
@tivv wdyt?